src/frontends/onnx/tests/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ foreach(BACKEND_NAME ${ACTIVE_BACKEND_LIST})
 endforeach()
 
 add_executable(ov_onnx_frontend_tests ${SRC})
+add_test(NAME ov_onnx_frontend_tests COMMAND ov_onnx_frontend_tests)
```

Comment:
I suggest adding labels.
Now, we have CPU, GPU, IE, may be others.

Reply:
Fixed